x-pack/plugins/security_solution/cypress/integration/detection_rules/export_rule.spec.ts

```diff
@@ -86,4 +88,51 @@ describe('Export rules', () => {
       `Successfully exported ${expectedNumberCustomRulesToBeExported} of ${totalNumberOfRules} rules. Prebuilt rules were excluded from the resulting file.`
     );
   });
+
+  context('rules with exceptions', () => {
+    before(() => {
+      deleteExceptionList(getExceptionList().list_id, getExceptionList().namespace_type);
+    });
+
+    beforeEach(() => {
+      // create rule with exceptions
+      createExceptionList(getExceptionList(), getExceptionList().list_id).then((response) =>
+        createCustomRule(
+          {
+            ...getNewRule(),
+            name: 'rule with exceptions',
+            exceptionLists: [
+              {
+                id: response.body.id,
+                list_id: getExceptionList().list_id,
+                type: getExceptionList().type,
+                namespace_type: getExceptionList().namespace_type,
+              },
+            ],
+          },
+          '2'
+        )
+      );
+    });
+
+    it('exports custom rules with exceptions', function () {
+      // one rule with exception, one without it
+      const expectedNumberCustomRulesToBeExported = 2;
+
+      loadPrebuiltDetectionRulesFromHeaderBtn();
+
+      selectAllRules();
+      bulkExportRules();
+
+      // should display correct number of custom rules when one of them has exceptions
+      cy.get(MODAL_CONFIRMATION_BTN)
+        .should('have.text', `Export ${expectedNumberCustomRulesToBeExported} Custom rules`)
```

Comment:
Nit: `Export ${expectedNumberCustomRulesToBeExported} custom rules`. But looks like it's case-insensitive.

Reply:
This PR was created before changes to button label(when Custom was suggested to make lowercase).
Thus, capitalized version here. After PR with changes was merged, I've updated this test as well

Reply:
Oh, I see! 👍